package/CHANGELOG

```diff
@@ -39,6 +39,8 @@ Enhancements
   * Added computation of Mean Squared Displacements (#2438, PR #2619)
   * Improved performances when parsing TPR files (PR #2804)
   * Added converter between Cartesian and Bond-Angle-Torsion coordinates (PR #2668)
+  * Improved performance of select_atoms on strings (e.g. name, type, resname) and
```

Comment:
I’d also put that under Fixes because it fixes the performance regression. (Having other faster things is a nice side-effect).